src/GraphQL/Execution/DocumentExecuter.cs

```diff
@@ -307,15 +319,6 @@ protected virtual ExecutionContext BuildExecutionContext(ExecutionOptions option
         /// for example it can be SubscriptionExecutionStrategy from GraphQL.SystemReactive.
         /// </summary>
         protected virtual IExecutionStrategy SelectExecutionStrategy(ExecutionContext context)
```

Comment:
I think we don't need this method anymore. `ExecutionStrategySelector` is a right place to reconfigure decision.

Reply:
It is, but that doesn't mean this method isn't proper.  `DocumentExecuter` should have virtual methods as necessary to split up functionality contained within and to allow overriding functionality as desired.

Reply:
One could change:
1. Selector component
2. Registration order, add registrations , etc.

So I'm sure this method excessive. Extension points are already enough.

Reply:
Fine; it's excessive.

Reply:
It's also backwards compatible with any existing code.  It's also only two lines of code.  It's also good separation of logic.  It also doesn't hurt anything.

Reply:
If the task can be solved using one extension point instead of two, then I will prefer the first. We can make another 33 virtual methods, spreading them evenly throughout the code. Theoretically, they will not harm, but in practice there will be only unnecessary burden.

Reply:
Well, it is pretty easy to replace if necessary.  A derived class can implement `IExecutionStrategySelector` and use `this` when calling the base's constructor.  Mainly I wouldn't just because of all the reasons I outlined before, mostly because it makes existing code compatible without changes, and there is no reason to remove it.  I know you don't, but I think a very large percentage of users use the serial execution strategy for queries, due to the use of scoped database engines like EF.

Reply:
So for 2 lines of code, that don't interfere with anything, are not outdated, work well with the new enhancements, and all that, I wouldn't change it.  Prevents a breaking change.  I know it's just one less breaking change, but it's still one less breaking change, and one that I think will affect a large number of people.

Reply:
I give up :)

Reply:
If you insist, I will remove it.  I really wouldn't.  Removing a single method, containing one line of code, just isn't worth it here for a breaking change.

If we want to, in a separate PR, clean up most all the constructors in `DocumentExecuter` and along with it, this method, then I think that makes sense.